test/test_mmnet.py

```diff
@@ -341,3 +349,75 @@ def test_propagation_multiple_events_multiple_iteration(self):
         self.assertEqual(event_1.relevance, expected_relevance_1)
         self.assertEqual(event_2.relevance, expected_relevance_2)
         self.assertIs(mmnet.events[0], event_1)
+
+
+class TestEventsImport(unittest.TestCase):
+
+    def check_ego_network(
+            self, expected_number_of_nodes, expected_number_of_edges,
+            expected_number_of_circles, expected_number_of_events):
```

Comment:
So we have this exact method twice now, maybe there could be a shared network checker, which can be used in the tests.

Reply:
The methods are supposed to be different: the import people one  should just check the nodes and edges in the PersonMode and PersonToPerson  Crossnet, wheras  the other should check for events in the EventMode and EventToPersonCrossnet. I open an issue and fix it. 

The reason why there should not be a single function is that you are breaking modularity in testing checking for thinks that in those particular tests are blackboxes to you.